packages/builder/src/stores/builder/components.js

```diff
@@ -555,7 +575,9 @@ export class ComponentStore extends BudiStore {
       const patchResult = patchFn(component, screen)
 
       // Post processing
-      const migrated = this.migrateSettings(component)
+      const migrated = null
+
+      this.migrateSettings(component)
```

Comment:
Niiice catch 🙏 . I side-stepped this during development to avoid losing sample data when the screen was updated. Definitely should be set accordingly!